[CIT-378] collectd mapper documentation (#274)
thin-edge/thin-edge.io: docs/src/references/bridged-topics.md

 * `az/messages/devicebound/#` - Use this topic to subscribe for the messages that were sent from cloud to device.
 Any message published by Azure on one the subtopics of `devices/{device_id}/messages/devicebound/#`
 is republished here.
 
 
## Collectd topics

> When the [device monitoring feature is enable](../tutorials/device-monitoring.md),

Reviewer: ```suggestion
When the [device monitoring feature is enabled](../tutorials/device-monitoring.md),
```

Author: fixed

> monitor metrics are emitted by `collectd` on a hierarchy of MQTT topics.

Reviewer: ```suggestion
monitoring metrics are emitted by `collectd` on a hierarchy of MQTT topics.
```

Author: Fixed

* `collectd/$HOSTNAME/#` - All the metrics collected on the device (which hostname is `$HOSTNAME`).
* `collectd/$HOSTNAME/$PLUGIN/#` - All the metrics collected by a given collectd plugin, named `$PLUGIN`.

> * `collectd/$HOSTNAME/$PLUGIN/$METRIC` - The topic for a given metric, named `METRIC`.

Reviewer: ```suggestion
* `collectd/$HOSTNAME/$PLUGIN/$METRIC` - The topic for a given metric, named `$METRIC`.
```
just to keep it consistent with the other lines

Author: fixed

>    All the measurements are given as a pair of a Unix timestamp in milli-seconds and a numeric value.

Reviewer: ```suggestion
All the measurements are published as a pair of a Unix timestamp in milli-seconds and a numeric value in the format `$TIMESTAMP:$VALUE`. For example, `1623155717:98.6`.
```

Author: Added

> The `collectd-mapper` daemon process ingests these measurements and emit translated messages

Reviewer: ```suggestion
The `collectd-mapper` daemon process ingests these measurements and emits translated messages to
```

Author: Fixed
